test-specifications/token-service/tokenBurnTransaction.md

```diff
@@ -0,0 +1,197 @@
+| 12      | Burns paused fungible tokens                                                   | tokenId=<PAUSED_FUNGIBLE_TOKEN_ID>, amount="1000000", commonTransactionParams.signers=[<PAUSED_FUNGIBLE_TOKEN_SUPPLY_KEY>]                | The token burn fails with an TOKEN_IS_PAUSED response code from the network.                        | N                 |
+| 13      | Burns an amount of 1,000,000 NFTs                                              | tokenId=<CREATED_NON_FUNGIBLE_TOKEN_ID>, amount="1000000", commonTransactionParams.signers=[<CREATED_NON_FUNGIBLE_TOKEN_SUPPLY_KEY>]      | The token burn fails with an INVALID_TOKEN_BURN_METADATA response code from the network.            | N                 |
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 64362,
+  "method": "burnToken",
+  "params": {
+    "tokenId": "0.0.15432",
+    "amount": "1000000",
+    "commonTransactionParams": {
+      "signers": [
+        "302E020100300506032B657004220420DE6788D0A09F20DED806F446C02FB929D8CD8D17022374AFB3739A1D50BA72C8"
+      ]
+    }
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 64362,
+  "result": {
+    "status": "SUCCESS",
+    "newTotalSupply": "9000000"
+  }
+}
+```
+
+### **Serial Numbers:**
+
+- The list of NFT serial numbers to burn.
+
+| Test no | Name                                        | Input                                                                                                                                                                              | Expected response                                                                            | Implemented (Y/N) |
+|---------|---------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-------------------|
+| 1       | Burns an NFT                                | tokenId=<CREATED_NON_FUNGIBLE_TOKEN_ID>, serialNumbers=[<NFT_SERIAL_1>], commonTransactionParams.signers=[<CREATED_NON_FUNGIBLE_TOKEN_SUPPLY_KEY>]                                 | The token burn succeeds and the token's treasury account no longer contains the first NFT.   | N                 |
+| 2       | Burns 3 NFTs                                | tokenId=<CREATED_NON_FUNGIBLE_TOKEN_ID>, serialNumbers=[<NFT_SERIAL_1>, <NFT_SERIAL_2>, <NFT_SERIAL_3>], commonTransactionParams.signers=[<CREATED_NON_FUNGIBLE_TOKEN_SUPPLY_KEY>] | The token burn succeeds and the token's treasury account no longer contains any NFTs.        | N                 |
```

Comment:
Suggestion, perhaps include a burn with 3 NFTs, one of which is deleted.
I'm not actually sure if that will succeed partially or fail entirely.

Reply:
By deleted do you mean already burned? I added a test to test burning 3 NFTs where one is already burned

Reply:
Yes, I meant already burned.